jupyterbook/_config.yml

```diff
@@ -0,0 +1,41 @@
+title: The U.S. Integrated Ocean Observing System (IOOS)
+author: IOOS 
+copyright: "2021"
+logo: images/IOOS-logo_little.png
+
+#only build files that are there in the table of contents. else defaults to building all files 
+only_build_toc_files: true
+
+launch_buttons:
+  notebook_interface: "classic" # The interface interactive links will activate ["classic", "jupyterlab"]
+  binderhub_url: "https://mybinder.org"
+  colab_url: "https://colab.research.google.com"
```

Comment:
@lohithmunakala can we remove this? I think that having many ways to run notebooks can be confusing. Colab is great for ML and/or when you need a GPU but most of the notebooks we have run OK on regular binder.

Reply:
Ok, will remove Colab for now. I Will keep a blank string there.

Reply:
@ocefpaf, the one thing I do like about google colab is it's easy to spin up and play with a notebook (I've been using it alot with some of the more off the shelf notebooks). With binder a user has to wait for the environment to build, which can take quite a while depending on the env. How much more overhead is it to enable this?

Reply:
@MathewBiddle I don't think the environment would be installed on a Colab notebook while compared to binder.